config/locales/en.yml

```diff
@@ -306,9 +306,6 @@ en:
     confirm_by_code:
       unable_to_verify_address: Unable to confirm email address
 
-    find_or_create_unclaimed_user:
```

Comment:
very glad to see these unclaimed users gone.. these were a real point of confusion and don't think they were being used properly...

Reply:
Well only the translation is gone because I feel like we don't need this :) The unclaimed users themselves are still used by the Tutor preview course feature. You can probably remove them in Fall of 2023